projects/addon-table/components/table/table.template.html

```diff
@@ -0,0 +1,82 @@
+<ng-container *tuiLet="entries$ | async as entries">
+    <thead tuiThead class="header">
+        <tr class="row">
+            <th
+                *ngFor="let key of columns; let index = index"
+                tuiResizableColumn
+                class="th"
+            >
+                <div
+                    class="head"
+                    [class.head_number]="data[0] && isNumber(data[0][key])"
+                >
+                    <ng-container
+                        *ngIf="heads | tuiFindDirective : key as cell else plain"
+                    >
+                        <button
+                            *ngIf="cell.tuiHeadSorter else cell.template"
+                            type="button"
+                            class="sort"
+                            [class.sort_sorted]="sortedKey === key"
+                            (click)="updateSorter(cell.tuiHeadSorter)"
+                        >
+                            <ng-container
+                                *ngTemplateOutlet="cell.template"
+                            ></ng-container>
+                            <tui-svg
+                                src="tuiIconSortDown"
+                                class="arrow"
+                                [class.arrow_rotated]="direction === -1"
+                            ></tui-svg>
+                        </button>
+                    </ng-container>
+                    <ng-template #plain>{{key}}</ng-template>
+                </div>
+            </th>
+        </tr>
+    </thead>
+    <tr *ngFor="let row of sorted" class="row">
```

Comment:
Need to add tbody and ability to use multiple tbodies with labels and expand/collapse, but need final design first